### PR TITLE
[AEA-1287] error msg fix on package import failed

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

error msg fix on package import failed

## Fixes

https://github.com/fetchai/agents-aea/issues/2191

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tests/test_components/test_loader.py [hunk 1]
```diff
@@ -101,9 +108,10 @@ def test_component_loading_module_not_found_error_framework_package_with_wrong_a
     ):
         with pytest.raises(
             AEAPackageLoadingError,
-            match="No module named packages.some_author; No AEA package found with author name 'some_author'",
-        ):
+            match="An error occurred while loading protocol an_author/a_protocol:0.1.0:",
+        ) as e:
             load_component_from_config(component_configuration)
+            assert "some_type" in str(e)
```
Comment (DavidMinarsch) [inline tests/test_components/test_loader.py:114]: Please check for a more complete error message (like on LHS); so we see in the tests what is expected.

Comment (solarw) [reply tests/test_components/test_loader.py:114]: it's the second check, just to see the most informative part of the specific exception

Comment (DavidMinarsch) [reply tests/test_components/test_loader.py:114]: I know, but for test writers and reviewers (us) the previous bigger output was more helpful when reading tests.

Comment (DavidMinarsch) [reply tests/test_components/test_loader.py:114]: Just have a longer string there so we can see what the full error message says (excluding the trace)

**Review by DavidMinarsch — APPROVED** [review]
LGTM

File: aea/components/loader.py [hunk 2]
```diff
@@ -80,6 +79,10 @@ def load_component_from_config(  # type: ignore
         _handle_error_while_loading_component_generic_error(configuration, e)
 
 
+class AEAPackageNotFound(Exception):
+    """Exception when failed toi import package, cause not exists."""
```
Comment (marcofavorito) [inline aea/components/loader.py:83]:
```suggestion
    """Exception when failed to import package, cause not exists."""
```

**Review by marcofavorito — APPROVED** [review]
LGTM

**Pull request merged**